src/routes/choice-exercises/WordChoiceExerciseScreen.tsx

```diff
@@ -15,12 +15,15 @@ interface WordChoiceExerciseScreenPropsType {
 const WordChoiceExerciseScreen = ({ navigation, route }: WordChoiceExerciseScreenPropsType): ReactElement | null => {
   const { extraParams } = route.params
   const { trainingSetId } = extraParams
-  const { data: documents, loading } = useLoadDocuments(trainingSetId)
+  const response = useLoadDocuments(trainingSetId)
+  const [documents, setDocuments] = useState<DocumentType[] | null>(null)
 
-  if (documents === null || loading) {
+  if (documents === null || response.loading) {
     return null
   }
 
+  useEffect(() => {}, [])
+
```

Comment:
```suggestion
```

Reply:
It should not be allowed to run hooks conditionally, i.e. after an if with a return statement. Do we miss a linting rule here?

Reply:
We have this linting rule, but the ci does not fail though. I will fix this in a separate ticket.